diff/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: See the Difference
+edit: true
+---
+
+These are the differentials or "diffs" of the various drafts presented at the Paris summit.  Here, if you click on the links below,  you can see drafts presented side-by-side and easily compare the difference between them.
+
+* TOC
+{:toc}
+
+## 23 October 2015 vs 4 December 2015
+
+This is the diff comparing the original draft of the 23rd October 2015 with the draft of the 4th December.
+
+<a href="{{ "/diff/23-oct-vs-4-dec" | prepend: site.baseurl }}">See the diff here</a> 
```

Comment:
We do not need the prepend stuff here. If you do not have then you could do simple markdown which is what i would recommend.